samples/star/src/commonMain/kotlin/com/slack/circuit/star/transition/PetSharedTransitionKeys.kt

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2024 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+package com.slack.circuit.star.transition
+
+interface SharedTransitionKey
```

Comment:
do you think this is something worth encouraging in docs/promoting the marker interface to a shipped artifact?